data/src/main/java/io/quarkus/qe/data/marshallers/RepositoryMarshaller.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.qe.data.marshallers;
+
+import io.quarkus.qe.data.RepositoryEntity;
+import io.quarkus.qe.model.Repository;
+
+public final class RepositoryMarshaller {
+
+    private RepositoryMarshaller() {
+
+    }
+
+    public static final Repository fromEntity(RepositoryEntity entity) {
```

Comment:
will you consider move on this public final static to an Interface?

Reply:
+1 changed to be a service instead of a helper (in the future, we could have an interface for this if needed)